src/pages/Details.vue

```diff
@@ -1,11 +1,16 @@
 <template>
   <div class="parent">
+
+    <!-- <script type="text/javascript">
+    const CaseNumber = 
+    </script> -->
     <div class="left">
       <div class="top">
-        <div class="title">Case EF 2020-0104</div>
+        <div class="title">Case #{{caseDetails.caseNumber}}</div>
+        <div v-bind="test" />
```

Comment:
remove

Reply:
I don't have this in my file, should be removed already